std/uni.d

```diff
@@ -7286,10 +7287,27 @@ if (isInputRange!Range && is(Unqual!(ElementType!Range) == Grapheme))
 }
 
 /// Ditto
-Range byCodePoint(Range)(Range range)
+auto byCodePoint(Range)(Range range)
 if (isInputRange!Range && is(Unqual!(ElementType!Range) == dchar))
 {
-    return range;
+    static struct Result
+    {
+        private Range _range;
+        @property bool empty() { return _range.empty; }
```

Comment:
dscanner would tell here that `@property` functions w/o parameters can be `const` since the aggregate state is not modified by the call.